### PR TITLE
feat: Support PMML & GraphDef & Caffe2 extract

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @gaocegege 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/shell/run.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ then
     case $format in
     PMML )
         java -Dconfig.file=/opt/openscoring/application.conf -jar /opt/openscoring/openscoring-server-executable-2.0.1.jar --port 8080 &
-        sleep 10s;
+        sleep 15s;
```
Comment (gaocegege) [inline scripts/shell/run.sh:52]: Why do we need sleep here?

Comment (simon-cj) [reply scripts/shell/run.sh:52]: Because it start a java process in the same container, need wait openscoring server running.

Comment (simon-cj) [reply scripts/shell/run.sh:52]: In Clever 1.6.0, using a pod to run openscoring server, mode analyzer wait pod running, and then it try three times, sleep 3s for each loop.

**Pull request merged**